ci-operator/step-registry/upi/install/openstack/inventory/upi-install-openstack-inventory-commands.sh

```diff
@@ -58,7 +62,7 @@ yq --yaml-output "$(cat <<-EOF
 	| .all.hosts.localhost.os_ingress_fip="$(<"${SHARED_DIR}/INGRESS_IP")"
 	| del(.all.hosts.localhost.os_bootstrap_fip)
 	EOF
-	)" '/var/lib/openshift-install/upi/inventory.yaml' > "${SHARED_DIR}/inventory.yaml"
+	)" > "${SHARED_DIR}/inventory.yaml"
```

Comment:
We also need to [udpate the CIDR of the IPv6](https://github.com/openshift/installer/blob/master/upi/openstack/inventory.yaml#L80). Right now this value defaults to `fd2e:6f44:5dd8:c956::/64` which is also the value of the CIDRs we have in the external network, once we try to add the new IPv6 subnet to the router set with that external network they will conflict. Perhaps you can update its value to `2001:db8:2222:5555::/64` instead?

Reply:
Right. Plan is to apply in install-config both networks - I'll try to mimic IPI configuration. Currently, script which generates install config ignores all networks besides the first one.